app-install-banner/useraction-banner/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<!--
+Copyright 2014 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<html>
+  <head>
+    <title>Web App Install Banner: Detect Install or Cancel Prompt Sample</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="manifest" href="./manifest.json">
+    <script>
+    window.addEventListener('load', function() {
+
+      var outputElement = document.getElementById('output');
+
+      navigator.serviceWorker.register('service-worker.js', { scope: './' })
+        .then(function(r) {
+          console.log('registered service worker');
+        })
+        .catch(function(whut) {
+          console.error('uh oh... ');
+          console.error(whut);
+        });
+       
+      window.addEventListener('beforeinstallprompt', function(e) {
+        outputElement.textContent = 'beforeinstallprompt Event fired';
+        
+        e.userChoice.then(function(choiceResult) {
+          
+          console.log(choiceResult.outcome);
+          
+          if(choiceResult.outcome == 'dismissed') {
+            console.log('User cancelled homescreen install');
+          }
+          else {
+            console.log('User added to homescreen');
+          }
+        });
+        
+      });
+      
+    });
+    </script>
+  </head>
+  <body>
+    <h1>Web App Install Banner: Detect User Install or Cancel of prompt sample</h1>
+    
+    <p>Available in <a href="https://www.chromestatus.com/features/4540065577435136">Chrome 42+ (for Android)</a></p>
+
+    <p>The web app install banner will prompt the user to add your web app to the users home screen.
+    It will only prompt when a number of criteria have been met:
+    </p>
+
+    <ul>
+      <li>The app uses a service worker</li>
+      <li>The site is using SSL</li>
+      <li>The app has a manifest declared</li>
+      <li>The manifest has a `short_name`, 144 pixel icon and a type of 'image/png'</li>
+    </ul>
+    
+    <p>In this example you will below that indicates the action of the user. Did they install it, or did they cancel the install.</p>
+    
+    <pre id=output></pre>
+
+    <p>To see if this example works, come back in another day and you will see the banner.</p>
```

Comment:
"in another day" implies that I can come back tomorrow and I'd see the banner. Given the delay between visits that's necessary to trigger the banner, I think we need to strongly push folks to chrome://flags if they want to try this out.